spotlight/src/main/java/com/takusemba/spotlight/shapes/RoundedRectangle.java

```diff
@@ -0,0 +1,45 @@
+package com.takusemba.spotlight.shapes;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.PointF;
+import android.support.annotation.RequiresApi;
+import android.view.View;
+
+@RequiresApi(21)
```

Comment:
i dont want this shape restricted to api level 21, isnt there any way to draw without this restriction?

Reply:
There is no native implementation before 21, so it needs to be made by other means for which I don't currently have the time.